### ISSUE TITLE
Notices with WP_DEBUG on 

### ISSUE THREAD

**Issue by EnigmaWeb** [EnigmaWeb/wp-catalogue#6]
Plugin throws a bunch of notices in WP Admin if WP_DEBUG is turned on:

```
Notice: Undefined variable: callback in /var/www/wp.local.lt/wp-content/plugins/wp-catalogue/index.php on line 22

Notice: Use of undefined constant WPCACHEHOME - assumed 'WPCACHEHOME' in /var/www/wp.local.lt/wp-content/plugins/wp-catalogue/index.php on line 161

Notice: Use of undefined constant WPC_STYLE - assumed 'WPC_STYLE' in /var/www/wp.local.lt/wp-content/plugins/wp-catalogue/index.php on line 50

Notice: Use of undefined constant WPC_SCRIPT - assumed 'WPC_SCRIPT' in /var/www/wp.local.lt/wp-content/plugins/wp-catalogue/index.php on line 53

Notice: Use of undefined constant WPC_STYLE - assumed 'WPC_STYLE' in /var/www/wp.local.lt/wp-content/plugins/wp-catalogue/index.php on line 81

Notice: Use of undefined constant WPC_SCRIPT - assumed 'WPC_SCRIPT' in /var/www/wp.local.lt/wp-content/plugins/wp-catalogue/index.php on line 85
```

WordPress version: 4.0.1, plugin version: 1.7.3

**Comment by EnigmaWeb** [comment]
How's this one going?

**Comment by Base29** [comment]
I have this in my to do list .. I have tried to replicate the issue on my development server but wasn't able to .. so will be testing the issue on the live site ... WPC Pro is high priority because lots of users are having issues with it ...

**Comment by Base29** [comment]
Fixed in 1.7.5
